src/structures/BaseGuildTextChannel.js

```diff
@@ -47,7 +47,11 @@ class BaseGuildTextChannel extends GuildChannel {
     }
 
     if ('nsfw' in data) {
-      this.nsfw = Boolean(data.nsfw);
+      /**
+       * If the guild considers this channel NSFW
+       * @type {boolean}
+       */
```

Comment:
The JSDoc for this property is now duplicated. It is a couple of lines above this.